### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by saifullah767**
# Navigation

- All the sections can navigate through nav.
- The date and time are displayed locally.
- Contact page add.

Note:

- I used /* eslint-disable no-plusplus */ because of increement error. And we have the freedom to use it.
- Page link is not working because of Upper case in Index.html file. Which will resolve after merge.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 /* eslint-disable no-plusplus */
```
Comment (brytebee) [inline script.js:1]: - [x] **Required:** Kindly fix linter errors, rather than disable them as they ensure a uniform codebase in the community.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @saifullah767 & @ONJoseph👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- The Awesome book's application has a Navigation bar, Books list, Add book form and Contact info.✔️
- Application functions properly.✔️
- All links work correctly.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
**_Your README file is not visible for review._**

- [ ] **Required:** Kindly write the appropriate project title in your README doc.
- [ ] **Required:** Update the details of **Author** as this is a collaborated project.
- [ ] **Required:** Fix the live demo link if you can.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (saifullah767) [reply script.js:1]: I think we have the freedom to use this. Because it seems like a bug.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Awesome Books
```
Comment (zainsadaqat) [inline README.md:4]: Hi, @saifullah767! 😄 
Great job by implementing the things asked by the previous Code Reviewer but there's still one thing remaining that is the **Live Project Link** which is still not working. Kindly fix this. 👍🏻

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Hello @saifullah767  😃 😄 

You're doing a great job on your project. We're proud of you. Good job so far! 
# 👏🏻 👏🏻 👏🏻 

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
Hi @ONJoseph  and @saifullah767 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

nodding-yes

## Optional suggestions

- [ ] N/A

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**